include/avif/avif.h

```diff
@@ -706,6 +710,39 @@ AVIF_NODISCARD AVIF_API avifBool avifGainMapMetadataFractionsToDouble(avifGainMa
 
 #endif // AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP
 
+// ---------------------------------------------------------------------------
+
+#if defined(AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM)
+// Sample Transforms are a HIGHLY EXPERIMENTAL FEATURE. The format might still
+// change and images containing a sample transform item encoded with the current
+// version of libavif might not decode with a future version of libavif.
+// Use are your own risk.
+// This is based on a proposal from the Alliance for Open Media.
+
+typedef enum avifSampleTransformRecipe
+{
+    AVIF_SAMPLE_TRANSFORM_NONE,
+    // Encode the 8 most significant bits of each input image sample losslessly
+    // into one base image. The remaining least 8 significant bits are encoded
+    // in a separate hidden image item. The two are combined at decoding into
+    // one image with the same bit depth as the original image.
+    // It is backward compatible in the sense that only the base image may be
+    // decoded (ignoring the hidden image item), leading to a valid image but
```

Comment:
Nit/optional: "only the base image may be decoded" could be misinterpreted as "only the base image is allowed to be decoded".

One way to make this clearer is to rephrase it as "it is possible to decode only the base image".